OsmIntegrator.Tests/Helpers/Base/BaseHelper.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Newtonsoft.Json;
+using OsmIntegrator.ApiModels;
+using OsmIntegrator.ApiModels.Auth;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OsmIntegrator.Tests.Helpers.Base
+{
+    public class BaseHelper
+    {
+        protected const string HttpAddressAndPort = "https://localhost:44388/";
+        protected HttpClient _client;
+
+
+        protected async Task<HttpResponseMessage> LoginAdminAsync()
+        {
+            var loginData = new LoginData
+            {
+                Email = "admin@abcd.pl",
+                Password = "12345678",
+            };
+            var jsonLoginData = JsonConvert.SerializeObject(loginData);
+            //var json = "{\"Email\":\"admin@abcd.pl\",\"Password\":\"12345678\"}";
+
+            var content = new StringContent(jsonLoginData, Encoding.UTF8, "application/json");
+
+            var response = await _client.PostAsync("/api/Account/Login", content);
```

Comment:
Te ścieżki można sobie w jakimś constansie trzymać, bo pewnie będą się powtarzać między testami.

Reply:
Nie powtarza się, i chyba nie będzie. Jak się zacznie powtarzać to dodam const'a.

Reply:
OK